guide/guide.md

```diff
@@ -101,6 +101,45 @@ Do not put any spaces around the `.`.
 x.foo().bar().baz(x, y, z);
 ```
 
+### Match
+
+For a single-expression matcher to a single-expression matchee, which fits on
```

Comment:
I'm confused by this sentence. What are single-expression matchers and matchees?